chaoscenter/authentication/pkg/services/auth_config_service.go

```diff
@@ -0,0 +1,25 @@
+package services
+
+import (
+	"context"
+
+	"github.com/litmuschaos/litmus/chaoscenter/authentication/pkg/authConfig"
+)
+
+type authConfigService interface {
+	CreateConfig(config authConfig.AuthConfig) error
+	GetConfig(key string) (*authConfig.AuthConfig, error)
+	UpdateConfig(ctx context.Context, key string, value interface{}) error
+}
+
+func (a applicationService) CreateConfig(config authConfig.AuthConfig) error {
+	return a.authConfigRepo.CreateConfig(config)
+}
+
+func (a applicationService) GetConfig(key string) (*authConfig.AuthConfig, error) {
+	return a.authConfigRepo.GetConfig(key)
+}
+
+func (a applicationService) UpdateConfig(ctx context.Context, key string, value interface{}) error {
```

Comment:
I don't think this will be ever used

Reply:
Have kept it for future use in case we need to add more such cases

Reply:
We can always add it back if needed. As they say "developers spend more time reading the code than writing". We can keep it if you can think of any use case in the future where this can be utilised, but if not then this can be removed.